solutions/client-setup/namespaces/client-name-networking.yaml

```diff
@@ -76,6 +76,23 @@ spec:
   role: roles/dns.admin
   member: "serviceAccount:client-name-networking-sa@client-management-project-id.iam.gserviceaccount.com" # kpt-set: serviceAccount:${client-name}-networking-sa@${client-management-project-id}.iam.gserviceaccount.com
 ---
+# Grant GCP role Tier2 DNS Record Admin to GCP SA on Core DNS Project
+apiVersion: iam.cnrm.cloud.google.com/v1beta1
+kind: IAMPolicyMember
+metadata:
+  name: client-networking-sa-tier2-dns-record-admin-permission # kpt-set: ${client-name}-networking-sa-tier2-dns-record-admin-permission
```

Comment:
minor detail, the name doesn't exactly match what's in the setter:
`client-networking-sa-tier2-dns-record-admin-permission`
should be:
`client-name-networking-sa-tier2-dns-record-admin-permission`

Reply:
Good catch Alain. Addressed. Please verify.